code/modules/admin/topic.dm

```diff
@@ -1760,7 +1760,7 @@
 		P.faxmachineid = fax.UID()
 		P.loc = fax.loc // Do not use fax.receivefax(P) here, as it won't preserve the type. Physically teleporting the fax paper is required.
 		if(istype(H) && H.stat == CONSCIOUS && (istype(H.l_ear, /obj/item/radio/headset) || istype(H.r_ear, /obj/item/radio/headset)))
-			to_chat(H, "Your headset pings, notifying you that a reply to your fax has arrived.")
+			to_chat(H, "<span class = 'specialnotice'>Your headset pings, notifying you that a reply to your fax has arrived.</span>")

```

Comment:
this isnt working and I have no idea why

Reply:
still broken

Reply:
Discord @'d you.